.github/workflows/osrm-backend.yml

```diff
@@ -327,74 +341,74 @@ jobs:
 
           - name: gcc-7-release-shared
             continue-on-error: false
-            node: 12
+            node: 14
             runs-on: ubuntu-20.04
             BUILD_TOOLS: ON
             BUILD_TYPE: Release
             BUILD_SHARED_LIBS: ON
             CCOMPILER: gcc-7
             CXXCOMPILER: g++-7
 
-          - name: node-12-conan-linux-release
+          - name: node-14-conan-linux-release
             build_node_package: true
             continue-on-error: false
-            node: 12
+            node: 14
             runs-on: ubuntu-20.04
             BUILD_TYPE: Release
             CCOMPILER: clang-6.0
             CXXCOMPILER: clang++-6.0
             ENABLE_CONAN: ON
             NODE_PACKAGE_TESTS_ONLY: ON
 
-          - name: node-12-conan-linux-debug
+          - name: node-14-conan-linux-debug
             build_node_package: true
             continue-on-error: false
-            node: 12
+            node: 14
             runs-on: ubuntu-20.04
             BUILD_TYPE: Debug
             CCOMPILER: clang-6.0
             CXXCOMPILER: clang++-6.0
             ENABLE_CONAN: ON
             NODE_PACKAGE_TESTS_ONLY: ON
 
-          - name: node-14-conan-linux-release
+
+          - name: node-16-conan-linux-release
             build_node_package: true
             continue-on-error: false
-            node: 14
+            node: 16
             runs-on: ubuntu-20.04
             BUILD_TYPE: Release
             CCOMPILER: clang-6.0
             CXXCOMPILER: clang++-6.0
             ENABLE_CONAN: ON
             NODE_PACKAGE_TESTS_ONLY: ON
 
-          - name: node-14-conan-linux-debug
+          - name: node-16-conan-linux-debug
             build_node_package: true
             continue-on-error: false
-            node: 14
+            node: 16
             runs-on: ubuntu-20.04
             BUILD_TYPE: Debug
             CCOMPILER: clang-6.0
             CXXCOMPILER: clang++-6.0
             ENABLE_CONAN: ON
             NODE_PACKAGE_TESTS_ONLY: ON
 
-
-          - name: node-16-conan-linux-release
```

Comment:
We removed Node 12 jobs, but added Node 18 instead(which we implicitly had before, but named it "LTS")